### PR TITLE
Quality: PowerShell Summit GH-3330

### LINKED ISSUE #3330: Quality: Appx Module style guide fixes
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples, Formatting command syntax, Markdown linting

### Article List

Appx

### PR DESCRIPTION
**Pull request by ThomasNieto**
# PR Summary

- Fixes #3330

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/appx/Add-AppSharedPackageContainer.md [hunk 1]
```diff
@@ -18,26 +19,29 @@ Add-AppSharedPackageContainer [-Path] <String> [-ForceApplicationShutdown] [-Mer
 ```
 
 ## DESCRIPTION
-The Add-AppSharedPackageContainer cmdlet deploys the shared package container definiton for the particular user.
+
+The Add-AppSharedPackageContainer cmdlet deploys the shared package container definition for the
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/appx/Add-AppSharedPackageContainer.md:23]: We want to make sure cmdlet names are wrapped in backticks, per the [style guide][01], so they render correctly and reliably.

```suggestion
The `Add-AppSharedPackageContainer` cmdlet deploys the shared package container definition for the
```

[01]: https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.3

File: docset/winserver2022-ps/appx/Add-AppxVolume.md [hunk 2]
```diff
@@ -20,28 +21,32 @@ Add-AppxVolume [-Path] <String[]> [-WhatIf] [-Confirm] [<CommonParameters>]
...
-This cmdlet returns the volume that it adds.
-Note, the *Path* parameter must be specified as a drive letter followed by "WindowsApps" as the directory.
-Not using the aforementioned format could lead to inconsistent behavior in the application model subsystems or the volume itself; for more information see the examples section.
+
+The **Add-AppxVolume** cmdlet adds an **AppxVolume** for the Package Manager to advertise. After you
+add a volume, appx deployment operations can use that volume as a target. This cmdlet returns the
+volume that it adds. Note, the *Path* parameter must be specified as a drive letter followed by
+"WindowsApps" as the directory. Not using the aforementioned format could lead to inconsistent
+behavior in the application model subsystems or the volume itself; for more information see the
+examples section.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/appx/Add-AppxVolume.md:30]: We always bold parameter names. If a value in the prose is specific for entry in a commandline, we want to backtick those values too.

```suggestion
The `Add-AppxVolume` cmdlet adds an **AppxVolume** for the Package Manager to advertise. After you
add a volume, appx deployment operations can use that volume as a target. This cmdlet returns the
volume that it adds. Note, the **Path** parameter must be specified as a drive letter followed by
`WindowsApps` as the directory. Not using this format could lead to inconsistent behavior in the
application model subsystems or the volume itself; for more information see the examples section.
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Thanks so much for this huge contribution, @ThomasNieto! I noticed some formatting issues around cmdlet names and properties. I added a few example suggestions, but if you could update the PR so that the same rules are applied across the files, that would be great!

**Review by sdwheeler — APPROVED** [review]
LGTM

**Pull request merged**